### PR TITLE
Some misc things

### PR DESCRIPTION
**Pull request by hawkowl**

### CODE REVIEW

**Comment by hawkowl** [comment]
cc @oberstet 
this will fix up #111 (unless there's any edge cases I'm unaware of, but it should catch them all)
it also improves coverage/the code of a bit of the CLI stuff -- not much to go until it's fully covered, a lot of it is low hanging fruit as far as coverage % goes, and it'll be quite rock solid at that point.

**Comment by hawkowl** [comment]
This also brings the 'hello' template in line with the one from the examples (using the new logging system).

File: tox.ini [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
+ignore = E501
+exclude = crossbar/worker/test/examples/syntaxerror.py
 
 [testenv:flake8]
 skip_install = True
 deps =
    flake8
 commands =
-   flake8 --ignore=E501 crossbar
+   flake8 crossbar
```
Comment (oberstet) [inline tox.ini]: how can that work since we do have long lines in the codebase?

Comment (hawkowl) [inline tox.ini]: Flake8 looks at the [flake8] section in tox.ini automatically -- so we don't need to define it again. This means we can run flake8 manually and it'll pick up the settings.

Comment (hawkowl) [inline tox.ini]: (from http://flake8.readthedocs.org/en/latest/config.html#per-project )

**Comment by oberstet** [comment]
@hawkowl ah, right. thanks for explaining!

**Pull request merged**